test_runner/src/test/kotlin/ftl/args/yml/test_error_yaml_cases/flank-no-device-version.yml

```diff
@@ -18,7 +18,7 @@ gcloud:
       orientation: portrait
       # Samsung Galaxy S9 SM-G9600
     - model: starqltechn
-      versxion: 28
+      version_: 28
```

Comment:
again, it's helpful to show "yes, we're trying not to use the real word" vs "oops, we didn't mean to make a typo, but here it is".